### PR TITLE
Item shroud

### PR DESCRIPTION
**Pull request by Declow0**
Issue #121

### CODE REVIEW

File: game/scripts/vscripts/items/shroud.lua [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	local shroud_duration = self:GetSpecialValueFor( "duration" )
+
+	EmitSoundOn( "Item.GlimmerCape.Activate", hTarget )
+	hTarget:AddNewModifier( hTarget, self, "modifier_ghost_state", { duration = shroud_duration } )
+	hTarget:AddNewModifier( hTarget, self, "modifier_item_glimmer_cape_fade", { duration = shroud_duration } )
+end
+
+--------------------------------------------------------------------------------
+
+item_shroud_2 = class({})
```
Comment (chrisinajar) [inline game/scripts/vscripts/items/shroud.lua]: `item_shroud_2 = item_shroud` would do the same thing and be cleaner

Comment (Declow0) [inline game/scripts/vscripts/items/shroud.lua]: ok. I don't know this...

Comment (Declow0) [inline game/scripts/vscripts/items/shroud.lua]: Done.

Comment (chrisinajar) [inline game/scripts/vscripts/items/shroud.lua]: Yeah, it's super useful for our upgrading items since the tiers always function the same.

**Pull request merged**